### ISSUE TITLE
Placeholders don't work

### ISSUE THREAD

**Issue by svenkatdsbe** [mkloubert/vs-deploy#110]
My config:

```
    "deploy": {
        "packages": [
            {
                "files": [
                    "index.php"
                ],
                "exclude": [
                    ".git", ".vscode"
                ]
            }
        ],
        "values": [
            {
                "name": "test",
                "type": "static",
                "value": "sk"
            }
        ],
        "targets": [
            {
                "type": "sftp",
                "user": "dsb",
                "password": "--changed--",
                "host": "10.0.76.2",
                "dir": "/home/dsb/samba_webroot/${test}"
            }
        ]
    }
```

My output:
```
Stelle Paket bereit...
Stelle '/index.php' in '/home/dsb/samba_webroot/${test}' bereit... [OK]
Fertig
```

Expected output:
```
Stelle Paket bereit...
Stelle '/index.php' in '/home/dsb/samba_webroot/sk' bereit... [OK]
Fertig
```

Created folder on target is *${test}* instead of *sk*
```
dsb@PHPdev:~/samba_webroot$ pwd
/home/dsb/samba_webroot
dsb@PHPdev:~/samba_webroot$ ls
${test}
dsb@PHPdev:~/samba_webroot$
```

Environment:
Windows 10 pro (10.0.15063) 64bit
VSC 1.16.1
Deploy 9.32.6

**Comment by mkloubert** [comment]
@svenkatdsbe

The `dir` property currently does not support this ... but I think it is a good idea to implement this.

I will do this that evening.

**Comment by mkloubert** [comment]
@svenkatdsbe 

**I released a new version 9.33.0**, which is now able to support placeholders in `dir` property.

**Comment by svenkatdsbe** [comment]
perfect. thank you
